crates/sozo/ops/src/migrate/mod.rs

```diff
@@ -168,11 +168,8 @@ where
                     // The injection of class hash and addresses is no longer supported since the
                     // world contains an internal DNS.
                     let args = if let Some(args) = init_call_args {
```

Comment:
The comment can be updated where we can remove that "only felts are supported".
We can keep the comment related to the `DNS`, since injection of other contract addresses are not yet supported again.